app/assets/javascripts/table-sort.es6

```diff
@@ -1,18 +1,33 @@
 $(document).ready(function() {
   if ($('table.sortable')) {
     let table;
+    
     $('th.sort')
+      .addClass("js-sortable")  //this class used to style as links, if JS not enabled, titles won't look clickable.
+      .attr('tabindex', 0)
       .wrapInner('<span class="sortable-column" title="sort this column"/>')
       .each(function(index) {
         const th = $(this),
           thIndex = index + 1;
         let inverse = false;
-
+        const endText = {
+          asc: {
+            numeric: ' from big to small',
+            alphabetic: ' in <em>reverse</em> order',
+            undefined: ' in <em>ascending</em> order' //for when the data-sort-type is not set
+          },
+          desc: {
+            numeric: ' from small to big',
+            alphabetic: ' in <em>alphabetical</em> order',
+            undefined: ' in <em>descending</em> order' //for when the data-sort-type is not set
+          }
+        }
```

Comment:
I'd move the definition of `endText` outside the `each` loop - as it is the same for every iteration - and only needs to be defined once. So put it straight after `let table;`